### DISCUSSION TITLE
Snapshot BSC Network

### DISCUSSION THREAD

**Discussion by alfirus** [snapshot-labs/snapshot#2847] [Q&A]
Hi,

I can see in https://snapshot.org/#/ripae.eth/proposal/bafkreihgeprxjmphh6yewklrypya77xx6splgtbog6ve6zxuaclp2oakge there are:

- PAE Fantom
- PAE Avax
- PAE Polygon
- PAE Cronos
- PAE DevFund

Why no PAE BSC?

**Comment by ChaituVR** [comment]
This has nothing to do with snapshot @alfirus You may have to contact `ripae.eth` space owner or admins to add support for this, If they have any trouble adding it to snapshot settings, we can help.
